websites/K/Kickstarter/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Kickstarter",
+	"description": {
+		"en": "Kickstarter exists to help bring creative projects to life. A home for film, music, art, theater, games, comics, design, photography, and more.",
+        "de": "Kickstarter existiert, um kreativen Projekten zur Verwirklichung zu verhelfen. Hier findest du Filme, Musik, Kunst, Theater, Spiele, Comics, Design, Fotografie und vieles mehr.",
+		"fr": "Kickstarter existe pour faire vivre les projets créatifs. Un site dédié à la musique, à l'art, au cinéma, au théâtre, à la BD, au design, à la photographie et aux jeux, entre autres.",
+        "es": "Kickstarter existe para ayudar a hacer realidad proyectos creativos. Un lugar para el cine, la música, el arte, el teatro, los juegos, las historietas, el diseño, la fotografía y más.",
+        "it": "Kickstarter esiste per aiutare a dare vita ai progetti creativi. Kickstarter accoglie tanti progetti: film, musica, arte, teatro, videogiochi, fumetti, grafica, fotografia e tanto altro.",
+        "ja_JP": "Kickstarterは、映画、音楽、アート、シアター、ゲーム、コミック、デザイン、写真など、多種多様なクリエイティブプロジェクトを実現させるための場です。"
```

Comment:
Formatting has seemed to messed itself up here; it is a combination of tabs and spaces

Reply:
I'll see to resolve that!

Reply:
Resolved in 1fdbed86bf8344a15b1a6430d2b4aa244e35ee9f